templates/base.html.twig

```diff
@@ -10,6 +10,7 @@
 
             <!-- Renders a link tag (if your module requires any CSS)
                  <link rel="stylesheet" href="/build/app.css"> -->
+        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.3/css/all.css" integrity="sha384-SZXxX4whJ79/gErwcOYf+zWLeJdY/qpuqC4cAa9rOGUstPomtqpuNWT9wdPEn2fk" crossorigin="anonymous">

```

Comment:
I think it's better to move third party font inclusions like this somewhere else, for instance in the styles.
We are using SCSS, so try importing the font by using @import, as demonstrated here:
https://stackoverflow.com/a/37373192